include/deal.II/grid/tria_accessor.h

```diff
@@ -924,6 +924,8 @@ class TriaAccessor : public TriaAccessorBase<structdim, dim, spacedim>
    *
    * This function is really only for internal use in the library unless you
    * absolutely know what this is all about.
+   *
+   * This function queries ReferenceCell::standard_vs_true_line_orientation().
```

Comment:
Why do we need this info?

Reply:
I probably don't *need* it, but I thought it would be useful to make the context of that function clearer by pointing to another place that explains what we are trying to do.